ha/core/error.py

```diff
@@ -249,3 +249,15 @@ def __init__(self, desc=None):
         _message_id = HA_EVENT_ANALYZER_ERROR
         _rc = 1
         super(EventAnalyzerError, self).__init__(rc=_rc, desc=_desc, message_id=_message_id)
+
+class ConfigureStonithResourceError(SetupError):
```

Comment:
Move to https://github.com/Seagate/cortx-ha/blob/node_status/ha/setup/setup_error.py

Reply:
Done.